test/async-hooks/test-graph.tls-write.js

```diff
@@ -66,8 +66,9 @@ function onexit() {
         id: 'tcpconnect:1', triggerAsyncId: 'tcp:1' },
       { type: 'TCPWRAP', id: 'tcp:2', triggerAsyncId: 'tcpserver:1' },
       { type: 'TLSWRAP', id: 'tls:2', triggerAsyncId: 'tcpserver:1' },
-      { type: 'Immediate', id: 'immediate:1', triggerAsyncId: 'tcp:2' },
-      { type: 'Immediate', id: 'immediate:2', triggerAsyncId: 'tcp:1' },
+      // TODO(ronag): Order is opposite on UNIX vs Windows.
+      // { type: 'Immediate', id: 'immediate:1', triggerAsyncId: 'tcp:2' },
+      // { type: 'Immediate', id: 'immediate:2', triggerAsyncId: 'tcp:1' },
```

Comment:
Not sure if this is significant.

Reply:
This boils down to the order of the `'close'` event on the server vs client socket instance.

Reply:
Seems related to the other `onStreamRead` behavior mismatch. Basically on the previously mentioned platforms the `onStreamRead` callback ending the readable client side of the stream is called before the handle.close() callback on the server side.